rmf_task/test/integration/test_backups.cpp

```diff
@@ -73,8 +113,10 @@ SCENARIO("Back up to file")
   // ====== Backup the task ======
   mock_active_task->issue_backup();
 
-  // ====== Restore the task ======
+  //// ====== Restore the task ======
   rmf_task::BackupFileManager restore(backup_root_dir);
+  restore.clear_on_startup(false); // Default settings
+  restore.clear_on_shutdown(true); 
```

Comment:
Is there a reason we're setting these flags when they're already the default?

Reply:
i implemented a naive `cleanup` function here https://github.com/open-rmf/rmf_task/commit/7bb5cad604d4eba8b8ce12ac265f882cfa3b6856#diff-8699d7f254e3fbb7374e8dcc0375624ccd2c584a5ac3d25b8ef5e97fc5557711R33 that deletes everything in the test folder for a "clean" test workspace state that is independent of any of our implementations. I think that flag should have no effect, in this case, since the workspace is always empty

Should we remove the cleanup function and restore that flag?

Reply:
Oh sorry, I didn't notice the cleanup function! Looks good to me :+1: